README.md

```diff
@@ -1,8 +1,34 @@
 # Tekton Tasks Operator
 
-Tekton Tasks Operator is Go operator which takes care about 
-deploying [Kubevirt tekton tasks](https://github.com/kubevirt/kubevirt-tekton-tasks) 
-and example pipelines.
+## Github
+https://github.com/ksimon1/kubevirt-tekton-tasks-operator (WIP)
+
+## Purpose
+deploy kubevirt tekton tasks (https://github.com/kubevirt/kubevirt-tekton-tasks) and example pipelines (ideas here https://docs.google.com/document/d/1q7pDGyGLB7KHghtecQrm3ktU_QLdjt0DDXXOqbwIpmw/edit#heading=h.ky0d3evm3f99)
+
+## Prerequisites
+Tekton - https://github.com/tektoncd
+KubeVirt - https://github.com/kubevirt/kubevirt
+CDI - https://github.com/kubevirt/containerized-data-importer
+
+## Operator framework
+https://sdk.operatorframework.io/
+This framework was chosen, because we have the most experiences with it, 
+it is easily modified, written in golang and consume small amount of resources.
+
+## Certification rotation
+https://github.com/qinqon/kube-admission-webhook 
+
```

Comment:
Is the plan to use the kube-admission-webhook directly?
Will the kube-admission-webhook  moved to kubevirt soon?